commit ea39aeca8a39b78bfecea2864c9e2a4cd8502249 (refs/changes/96/4396/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-06-22T09:24:50-07:00 (1 year, 4 months ago)
    
    TRITON-531 sdcadm checkIfTmpVMHasErrors calls back with a string instead of an Error instance
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/lib/procedures/shared.js b/lib/procedures/shared.js
index f20c70b..3d1dc85 100644
--- a/lib/procedures/shared.js
+++ b/lib/procedures/shared.js
@@ -614,15 +614,17 @@ function checkIfTmpVMHasErrors(arg, next) {
                 next(err);
                 return;
             }
-            var errs = stdout.trim();
-            if (errs) {
+            var errOutput = stdout.trim();
+            if (errOutput) {
                 if (counter < limit) {
                     setTimeout(_checkIfItHasErrors, 5000);
                     return;
                 } else {
                     progress('Timeout (5m) waiting for %s to be up',
                             arg.tmpUUID);
-                    next(errs);
+                    next(new errors.InternalError({
+                        message: errOutput
+                    }));
                     return;
                 }
             }
